bismuthcore/bismuthcore/rpcconnection.py

```diff
@@ -54,7 +54,7 @@ def _send(self, data, slen=SLEN, retry=True):
             self.sdef.settimeout(LTIMEOUT)
             # Make sure the packet is sent in one call
             sdata = str(json.dumps(data))
-            res = self.sdef.sendall(str(len(sdata)).encode("utf-8").zfill(slen)+sdata.encode("utf-8"))
+            _ = self.sdef.sendall(str(len(sdata)).encode("utf-8").zfill(slen)+sdata.encode("utf-8"))
```

Comment:
I would even remove the `_ = ` for better readability, or check res for added safety, make sure all was indeed sent.

Reply:
Definitely a check would be optimal. A few improvements could be made to this line altogether.